tests/sklearn/test_sklearn_autolog.py

```diff
@@ -1859,3 +1859,23 @@ def test_is_metrics_value_loggable():
     assert not is_metric_value_loggable(np.bool(True))
     assert not is_metric_value_loggable([1, 2])
     assert not is_metric_value_loggable(np.array([1, 2]))
+
+
+@pytest.mark.parametrize("log_eval_metrics", [True, False])
+def test_log_eval_metrics_configuration(log_eval_metrics):
```

Comment:
Can we verify that `log_eval_metrics` can be toggled on / off and add similar verification for `pyspark.ml`?

Reply:
@dbczumar You mean a test like https://github.com/mlflow/mlflow/pull/4705/files#diff-bd11e9a3dea4e2026366976e469085d6354fe088f0a933bc43a56177bc863376R763-R780?